hs_core/templates/400.html

```diff
@@ -94,9 +94,16 @@
                         <a href="#" class="dropdown-toggle" data-toggle="dropdown">{{ request.user }} <b class="caret"></b></a>
                         <ul class="dropdown-menu">
                             <li class="account">
-                                <p><b>{{ request.user }}</b>{% if request.user.email %}<br />{{ request.user.email }}{% endif %}</p>
-                                <p><a href="{% url 'profile_update' %}">Profile</a> | <a href="{% url 'logout' %}" id="signout-menu">Sign out</a>
-                                <div class="clearfix"></div>
+                                {% if request.user.userprofile.picture and request.user.userprofile.picture.url%}
+                                <div style="background-image: url('{{ request.user.userprofile.picture.url }}'); float:left;"
+                                     class="round-image profile-pic-thumbnail">
+                                </div>
+                                {% endif %}
```

Comment:
Does it looks right and layout correct with the photo missing? Maybe instead we should show a placeholder for profiles without a photo?